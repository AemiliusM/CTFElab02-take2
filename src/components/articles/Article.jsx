import React from 'react';

export default function Article({ title, author, description }) {
    return (
        <div>
            <h1>Title: {title}</h1>
            <span>Author: {author}</span>
            <h3>Description: {description}</h3>
        </div>
    );
}