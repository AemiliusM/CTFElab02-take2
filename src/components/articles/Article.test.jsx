import React, { Component } from 'react';
import { render } from '@testing-library/react';
import Article from './Article';

describe('article test yo!', () => {
    it('should render a single article', () => {
        const { asFragment } = render(
            <Article 
                title='Something, something, something, darkside'
                author='robot chicken'
                description='skit'
                />
        );
        expect(asFragment()).toMatchSnapshot();
    });
});