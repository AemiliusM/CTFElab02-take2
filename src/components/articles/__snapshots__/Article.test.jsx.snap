// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`article test yo! should render a single article 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Title: Something, something, something, darkside
    </h1>
    <span>
      Author: robot chicken
    </span>
    <h3>
      Description: skit
    </h3>
  </div>
</DocumentFragment>
`;
